### ISSUE TITLE
`<DESCRIBE::>` or `<IT::>` with no description of the block eats all stdout output for that block

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars-runner-cli#600]
This comes from a well-known JS test framework bug: If you put empty string as the first argument of `Test.describe`/`Test.it`, all stdout in the block, including test output and user logging statements, will be eaten. For example, if I wrote this as the test block:

```
Test.describe('', _=>{
  console.log('test');
  Test.assertEquals(1,1);
  Test.assertEquals(1,2);
  Test.it('inner', _=>console.log('test2'));
});
```

I would get *nothing* when running the tests, and the only thing I get is "Unhandled rejection TestError: Expected: 2, instead got: 1".

Similarly, if I do this:
```
Test.describe('outer', _=>{
  console.log('test');
  Test.assertEquals(1,1);
  Test.assertEquals(1,2);
  Test.it('', _=>console.log('test2'));
});
```

The stdout output from the `it` block is completely gone:

![image](https://user-images.githubusercontent.com/14909989/39120557-5ff2bce4-4721-11e8-8469-9a18945f8435.png)

I inspected the actual test output myself, and it's like this:

```
<DESCRIBE::>
test

<PASSED::>Test Passed: Value == 1

<FAILED::>Expected: 2, instead got: 1

<IT::>inner
test2

<COMPLETEDIN::>

<COMPLETEDIN::>4

<DESCRIBE::>outer
test

<PASSED::>Test Passed: Value == 1

<FAILED::>Expected: 2, instead got: 1

<IT::>
test2

<COMPLETEDIN::>

<COMPLETEDIN::>
```

So basically, if `<DESCRIBE::>` or `<IT::>` has nothing next to it and goes to a line break instantly, everything until the next `<COMPLETEDIN::>` is eaten.

I don't see the relevant handling code from the runner, so I guess it's handled by the CW backend itself? Then I presume only @jhoffner can fix this.

**Comment by kazk** [comment]
I'll look into this. The output parser can add generic test group name if it's empty.

**Comment by kazk** [comment]
I've fixed this but not deployed yet. I'll close this after deploying.
